editorextensions/codebeamer-custom-ui/modal/src/api/lucidGateway.ts

```diff
@@ -130,3 +137,22 @@ export function startImport(id: number, items: number) {
 export function closeModal() {
 	window.parent.postMessage({ action: 'closeModal', payload: true }, '*');
 }
+
+export function handleCardBlocks(callback: (arg0: any) => void) {
```

Comment:
JSDoc please. And the naming doesn't look quite sound to me.. `handleCardBlocks`, but you actually handle `getCardBlocks`, right? But what is `getCardBlocks` anyway? You only define that within the function itself, so there's no context on it soutside.